ana/management/commands/ana_test_data.py

```diff
@@ -0,0 +1,31 @@
+from ana.models import Verb, Record
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from random import randint
+
+from uuid import uuid4
+
+
+class Command(BaseCommand):
+    help = 'Generate test records for ana app'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-vb', '--verb', type=str, help='Verb slug for the records',
```

Comment:
@emanlodovice I believe the `-vb` is shorthand for `--verb`? I think it's good convention to use single characters for shorthands, especially because multiple shorthands can be combined into one, e.g. `-ab` === `-a -b`. `-vb` might be confused for `-v -b` as well.

Reply:
@arnellebalane got it. hmm but I don't think i can use `-v` thought because that's usually used for verbosity level.

Reply:
@emanlodovice Or version haha hmm. Perhabs `-s` is okay, since the value going to be passed is a `slug`?

Reply:
@arnellebalane i've updated the code in https://github.com/emanlodovice/ana/pull/31/commits/5aaa021b4a7fcbbbf4b95801fb01263ebd00be1b thanks for the review 👍